src/patternfly/demos/DataToolbar/examples/DataToolbar.css

```diff
@@ -0,0 +1,6 @@
+#ws-example-d-datatoolbar-data-toolbar-attribute-value-single-select-filter-desktop,
+#ws-example-d-datatoolbar-data-toolbar-attribute-value-single-select-filter-mobile,
+#ws-example-d-datatoolbar-data-toolbar-attribute-value-checkbox-select-filter-desktop,
+#ws-example-d-datatoolbar-data-toolbar-attribute-value-checkbox-select-filter-mobile {
+  min-height: 400px;
```

Comment:
@redallen for some reason these styles are applying locally but not in the build

Reply:
Yeah, this is our fault. Try using rebasing and using the prefix `ws-core` instead of `ws-example`. Sorry!